_posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2019-07-16
+title: AWS CloudFormation Example
+categories:
+   - AWS
+description: How to enable CloudFormation and deploy a simple template.
+type: Document
+---
+
+## Background
+
+This document will show you how to enable the CloudFormation stage.  This document also assumes that you have configured an AWS account.  If you need more instructions on configuring your AWS account please refer to either [Deploying to AWS from Spinnaker (using IAM instance roles)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account-iam/) or [Deploying to AWS from Spinnaker (using IAM credentials)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account/) 
+
+## Preconditions
+&nbsp;
+### Enable Cloudformation in Spinnaker
+
+First, we need to enable CloudDriver by adding a `clouddriver-local.yml` file to your hal config profiles directory e.g. `.hal/default/profiles/clouddriver-local.yml`.
+```yaml
+aws:
+  features:
+    cloudFormation:
+      enabled: true
+```
+You can check your configuration by running `hal deploy apply`.
+
+&nbsp;
+### Halconfig 
+
+Make sure to enable the aws provider and have set at least one region
+
+```yaml
+aws:
+  enabled: true
+  accounts:
+  - name: aws-1
+    requiredGroupMembership: []
+    providerVersion: V1
+    permissions: {}
+    accountId: '123456789123'
+    regions:
+    - name: us-east-1
+    - name: us-west-2
+    assumeRole: role/SpinnakerManagedRole
+  primaryAccount: aws-1
+  bakeryDefaults:
+    templateFile: aws-ebs-shared.json
+    baseImages: []
+    awsAssociatePublicIpAddress: true
+    defaultVirtualizationType: hvm
+  defaultKeyPairTemplate: '-keypair'
+  defaultRegions:
+  - name: us-west-2
+  defaults:
+    iamRole: BaseIAMRole
+```
+
+&nbsp;
+### CloudFormation permissions in Policy
+
+Another thing to check is the `cloudformation:*` permission in the *"Action"* section inside your **Policy**, like in the following example. 
```

Comment:
Add a comment that this needs to be done in the AWS console, and what section you should check

Reply:
also put the reference to the another doc